src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java

```diff
@@ -172,14 +180,29 @@
      * Transit search as well; Hence this is named search-window and not raptor-search-window. Do
      * not confuse this with the travel-window, which is the time between EDT to LAT.
      * <p>
-     * Use {@code null} to unset, and {@link Duration#ZERO} to do one Raptor iteration. The value is
-     * dynamically  assigned a suitable value, if not set. In a small to medium size operation
+     * Use {@code null} to unset, and {@link Duration#ZERO} to do one Raptor iteration. The value
+     * is dynamically  assigned a suitable value, if not set. In a small to medium size operation
      * you may use a fixed value, like 60 minutes. If you have a mixture of high frequency cities
      * routes and infrequent long distant journeys, the best option is normally to use the dynamic
      * auto assignment.
+     * <p>
+     * There is no need to set this when going to the next/previous page any more.
      */
     public Duration searchWindow;
 
+    /**
+     * Use the cursor to go to the next "page" of trips. You should pass in the original request as
+     * is.
+     * <p>
+     * At the moment we only support paging to the next page. If arriveBy is false, then page to
+     * the itineraries witch depart AFTER the current result. If arriveBy is true, then page to
+     * the itineraries witch arrive BEFORE the current result.
+     * <p>
+     * The paging do not support timeTableView=false and arriveBy=true, this will result in
+     * none pareto-optimal results.
```

Comment:
```suggestion
     * Use the cursor to go to the next or previous "page" of trips.
     * You should pass in the original request as is.
     * <p>
     * If arriveBy is false, then the next page of itineraries will depart AFTER the current results
     * and the previous page of itineraries will depart BEFORE the current results.
     * If arriveBy is true, then the next page of itineraries will arrive BEFORE the current results
      * and the previous page of itineraries will arrive AFTER the current results.
     * <p>
     * The paging do not support timeTableView=false and arriveBy=true, this will result in
     * none pareto-optimal results.
```